Makefile.am

```diff
@@ -121,6 +126,72 @@ CLEANFILES += $(eos_updater_avahi_in:%.in=%)
 
 dist_man8_MANS += eos-updater-avahi/docs/eos-updater-avahi.8
 
+# Server utility library
```

Comment:
I find that more confusing, since it splits a single evaluation scope across multiple files, so the included `Makefile.am`s end up using rules which are defined in a different file.